corrector.rb

```diff
@@ -0,0 +1,6 @@
+class Corrector
+  def correct_name(name)
+    name = name.size > 10 ? name[0, 10] : name
```

Comment:
- [ ] The extra logic for checking the size of the name would just increase the runtime of this method. This part => `name[0, 10]` already takes care of the whole logic for you. It will always trim if the characters in the name are more than ten or return the name as it is when the characters are less than 10. So refactoring as below would help improve the method a little:

  ```ruby
  name = name[0, 10]
  ```
  
  Another better approach that should save us on runtime is capitalizing the name in place first, then returning the trimmed version. For example:
  
  ```ruby
  def correct_name(name)
    name.capitalize!
    name[0,10]
  end
  ```

Reply:
Hi @Lameck1,
Thanks a lot for the suggestion! Definitely, I will implement it!